RequestService/RequestService.Handlers/GetNewsTickerHandler.cs

```diff
@@ -38,23 +39,23 @@ public async Task<NewsTickerResponse> Handle(NewsTickerRequest request, Cancella
                 {
                     Value = item.Value,
                     SupportActivity = item.SupportActivity,
-                    Message = $"**{ item.Value }** {item.SupportActivity.FriendlyNameShort()} { item.SupportActivity.RequestType().ToString() }  completed"
+                    Message = $"**{ item.Value }** *{item.SupportActivity.FriendlyNameShort().ToLower()}* { item.SupportActivity.RequestType().PerfectTense(Convert.ToInt32(item.Value)) } completed"
```

Comment:
Just wondering whether here we should be makeing use of the existing PerfectTense extension method for SupportActivities, which provides slightly more varied wording, e.g. "5 dogs walked", "6 errands run".

We would need to add the markdown formatting, but that could enhance where that method is used already (the recognition banner towards the top of the screen, when logged in.)